test/pred-sets/ref/ccb_quad.inv

```diff
@@ -30,6 +30,9 @@ sd::oec.weighted_labeled_examples 8
 current_pass 1
 l1_state 0
 l2_state 1
+gd_ppw_state.size() = 1
```

Comment:
we might be able to avoid breaking compat, if we put this extra state after the weights and not before